docs/installation.rst

```diff
@@ -0,0 +1,386 @@
+
+.. code-block:: bash
+
+    pyenv install -l
+
+You should see a very long list of versions to choose from. Let's install the latest version of the 3.9 family:
+
+.. code-block:: bash
+
+    pyenv install 3.9.13
+
+The command will take a minute or two to complete, depending on your internet connexion, and show an output similar to the following (this is just an example for Windows):
+
+.. code-block:: output
+
+    :: [Info] ::  Mirror: https://www.python.org/ftp/python
+    :: [Downloading] ::  3.9.13 ...
+    :: [Downloading] ::  From https://www.python.org/ftp/python/3.9.13/python-3.9.13-amd64.exe
+    :: [Downloading] ::  To   C:\Users\your_username\.pyenv\pyenv-win\install_cache\python-3.9.13-amd64.exe
+    :: [Installing] ::  3.9.13 ...
+    :: [Info] :: completed! 3.9.13
+
+Now, we have a new Python version in our system, but it is still not available (if you run ``python --version`` you will get the same result as before). There are two options moving forward:
+
+- If you want to set it as the global python version, available system wide (only do this if you really want to set is as your main Python!) run:
+
+    .. code-block:: bash
+
+        pyenv global 3.9.13
+
+- If you just want it momentarily to install MUSE run instead the following command:
+
+    .. code-block:: bash
+
+        pyenv shell 3.9.13
+
+In both cases, if you run ``python --version`` afterwards, you should get ``Python 3.9.13``.
+
+Installing ``pipx``
+~~~~~~~~~~~~~~~~~~~
+
+Next we need to install ``pipx``, a Python application manager that facilitates installing, keeping applications updated and run them in their own isolated environments. We could skip this step and install MUSE directly, but that will risk to have conflicting dependencies in the future if you install any other application, breaking your MUSE installation, and we do not want that to happen.
+
+The installation instructions for ``pipx`` can be found in the `official webpage <https://pypa.github.io/pipx/installation/>`_ specific for the three operative systems. The following instructions, however, should work for the three cases:
+
+.. code-block:: bash
+
+    python -m pip install pipx
+    python -m pipx ensurepath
+
+Make sure you run these commands with a compatible Python version, as described in the previous section. If for whatever reason, this does not work, follow the system specific instructions in the webpage.
+
+Installing MUSE itself
+~~~~~~~~~~~~~~~~~~~~~~
+
+With all the system prepared, installing MUSE is the easiest part:
+
+.. code-block:: bash
+
+    python -m pipx install muse-os
+
+As above, make sure you run this command with the appropriate Python version.
+
+And that is all! Now, MUSE should be available system wide simply by invoking ``muse`` in the terminal. To check that everything works as expected, open a new terminal and run:
+
+.. code-block:: bash
+
+    muse --model default
+
+This will run a default, example model, completing after reaching year 2050.
+
```

Comment:
I cannot see the image, for some reason, but I can add it, yes.

Reply:
Fixed in 1dba66a .

By the way, @ahawkes pointed out, and now I can see what he meant, that the default simulation reports a warning related to the constrains for wind. I don't know since when that warning is there, but I cannot see any reason why changing the installation process has anything to do with it. Do you have any suggestion on why the default model is reporting this warning? Was it intentional? 

```
    -- 2023-07-20 13:45:25 - muse.mca - WARNING
    Check growth constraints for wind.

    -- 2023-07-20 13:45:25 - muse.mca - INFO
    Finish simulation year 2050!
```

Reply:
The growth constrain on wind comes from the fact that wind is somehow outside the system boundaries as it is not "produced" by the system. Normally, we ca use "exclude_commodities" to suppress this warning when misleading.

Reply:
Either way is fine with me. I think that @ahawkes was just finding odd that an example simulation had a warning on constrains.